settlements/README.md

```diff
@@ -103,20 +108,10 @@ for more details.
 
 ## GDPR
 
-Settlements reports are available both with and without personal details of the customer.
-This is due to [GDPR](https://ec.europa.eu/info/law/law-topic/data-protection_en), and whether
-the data processor is Vipps or the merchant.
-
-Reports containing personal information requires a data processor agreement.
-Due to this, reports with personal information can only be downloaded by logging in with
-BankID on [portal.vipps.no](https://portal.vipps.no) and accepting the terms there.
-
-The Norwegian text from [portal.vipps.no](https://portal.vipps.no):
-
-_Velger du å laste ned rapporter med personinformasjon blir bedriften din selvstendig behandlingsansvarlig for personinformasjonen som lastes ned. Dette betyr at bedriften din er selv ansvarlig for å sikre etterlevelse av peronvernreglene. For mer informasjon, vennligst se Vipps [vilkår](https://www.vipps.no/vilkar/) for bruk._
+Vipps does not provide personal details of the customer to the merchant as part of settlement reports, as there is no legal basis for Vipps as a data processor under European Data Protection laws to do this for all payments.
```

Comment:
I still think we need to say that personal details are available on the portal.

>Reports containing personal information requires a data processor agreement.
 Due to this, reports with personal information can only be downloaded by logging in with
 BankID on [portal.vipps.no](https://portal.vipps.no) and accepting the terms there.

Reply:
But it's not in ecom context?

Reply:
Reports contain all types of payments AFAIK.